src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExceptionListenerPass.php

```diff
@@ -27,13 +29,18 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        // register the exception controller only if Twig is enabled and required dependencies do exist
-        if (!class_exists('Symfony\Component\ErrorRenderer\Exception\FlattenException') || !interface_exists('Symfony\Component\EventDispatcher\EventSubscriberInterface')) {
+        // to be removed in 5.0
```

Comment:
@yceruto can you please send a PR on master to cleanup this?

Reply:
See https://github.com/symfony/symfony/pull/33476